lib/paper_trail/multi.ex

```diff
@@ -18,8 +18,11 @@ defmodule PaperTrail.Multi do
   def insert(
         %Ecto.Multi{} = multi,
         changeset,
-        options \\ [origin: nil, meta: nil, originator: nil, prefix: nil]
+        options \\ [origin: nil, meta: nil, originator: nil, prefix: nil, index: ""]
       ) do
+    model = String.to_atom("model" <> options[:index])
+    version = String.to_atom("version" <> options[:index])
```

Comment:
Although I see that these variables wont clash with the model and version references below, it is still in my opinion better to name them differently such as `model_key`, `version_key`. 

Also it might be a better idea to not have these variables and have them in the options with default values being "model", and "version" instead.

Reply:
@izelnakri thanks to you for the library in the first place and for considering my PR.

You're right, it does sound better the naming you suggest, I'll change it shortly, but AFAIK (forgive me I'm a rookie in Elixir) I can't use the ^ for pattern matching with something like `options[:model]`, so I'll keep at least the model variable.

I also ran `mix format` but it didn't change anything.